buildkite-webhook-listener.py

```diff
@@ -268,4 +299,12 @@ def deploy_tarball(artifact_url, target_dir):
         ),
         flush=True,
     )
+    if arg_keep_versions is not None:
+        print(
+            "Keeping only previous %s versions." % ( arg_keep_versions ),
```

Comment:
Prefer to pass a tuple to %. `( foo )` is the same as `foo`. You need a trailing comma inside the parens to make it a tuple.

```suggestion
            "Keeping only previous %i versions." % (arg_keep_versions, ),
```